docs/FAQ.md

```diff
@@ -56,13 +56,13 @@ No
 
 ### What is the policy on supporting end-of-life(EOL) Ruby versions?
 
-We will target the latest the EOL Ruby, currently that is 1.9.3.
+We will target the latest the EOL Ruby, please check the link below to see current EOL version.
 
-### What versions or Ruby do you test against?
+https://www.ruby-lang.org/en/downloads/branches/
 
-1.9.3, 2.0, 2.1, 2.2
+### What versions or Ruby do you test against?
 
-Note: Linting is not done against 1.9.3
+We should always check non EOL versions. Support may not be added yet but ideally that is always true.
```

Comment:
How about...

We strive to test on all non-EOL versions. It can take time to add new versions to all plugins.